akd/src/directory.rs

```diff
@@ -666,7 +666,7 @@ mod tests {
         auditor::audit_verify,
         // BUG: Key history verification is failing on small-sized trees. It's being addressed in Issue #144
```

Comment:
If we're really solving this issue, we should remove notes to Issue #144 in the code comments